deepmd/entrypoints/main.py

```diff
@@ -162,6 +162,13 @@ def parse_args(args: Optional[List[str]] = None):
         default="out.json",
         help="The output file of the parameters used in training.",
     )
+    parser_train.add_argument(
+        "-mu",
```

Comment:
Maybe short name `mt` is better?

Reply:
I've never see the usage of `-` along with two letters. Does it really work? `-m` or `--mu` is more common.

Reply:
Yes, it works but lacking some features of the single-character short option and not a good practice.